src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -788,3 +788,16 @@ unique_ptr<Scene> svgSceneBuild(SvgNode* node, float vx, float vy, float vw, flo
 
     return root;
 }
+
+void applyBBoxFlag(const Scene* scene, float& vx, float& vy, float& vw, float& vh, float& w, float& h, BBoxFlag bboxFlag)
```

Comment:
The only reason I added this function is because I didn;t want to change the loader members (vx, vy, vw, vh) from the svgSceneBuild function. calling a dedicated function seems to be more readable, but still, it should be placed in some common space I think, but maybe you have some better idea in general how to deal with no viewBox info than bboxflag